src/domain/Car.js

```diff
@@ -1,20 +1,36 @@
 class Car {
-  static NAME_LENGTH = 5;
+  static CAR_MAX_NAME_LENGTH = 5;
+  static CAR_MIN_NAME_LENGTH = 1;
```

Comment:
```suggestion
  static NameRange = { MIN: 1,MAX: 5 }
```
이렇게 namespace로 묶어서 표현해줄 수도 있을 것 같네요!